packages/dispatchr/lib/DispatcherContext.js

```diff
@@ -37,7 +37,11 @@ DispatcherContext.prototype.getStore = function getStore(name) {
     if (!this.storeInstances[storeName]) {
         var Store = this.dispatcher.stores[storeName];
         if (!Store) {
-            throw new Error('Store ' + storeName + ' was not registered.');
+            if ('function' === typeof name) {
```

Comment:
A warning that the store may miss some dispatches? I think it would still leave the app in a potentially bad state, plus it would spam the user.